client/apis/recipes.js

```diff
@@ -0,0 +1,21 @@
+import request from 'superagent'
+
+// const rootUrl = '/api/v1'
+
+// TODO: create .env file to store my KEY
+// Adjust config/setup -- Dotenv
+
+export function getRecipes(userInput) {
```

Comment:
I would make this abit clearer about what this is meant to be. `userInput` doesnt tell me much about what this is going to be.

Reply:
@ZinHoang Could you please follow up on this one.